src/components/__tests__/hangman_game.test.jsx

```diff
@@ -0,0 +1,117 @@
+import { mount } from "enzyme";
+import HangmanGame from "../HangmanGame";
+import { act } from "react-dom/test-utils";
+
+describe("HangmanGame", () => {
+  const setGuesses = jest.fn();
+  const setFails = jest.fn();
+  const guesses = [];
+  const words = ["HELLO"];
+  const guessAttempts = 6;
+  const fails = 2;
+
+  const component = mount(
+    <HangmanGame
+      {...{
+        words,
+        guesses,
+        guessAttempts,
+        setGuesses,
+        fails,
+        setFails,
+      }}
+    />
+  );
+
+  describe("to play the game", () => {
+    it("initially needs to render the guessing component", () => {
+      expect(component.find("GuessInput")).toExist();
+      expect(component.find("GameOverScreen")).not.toExist();
+    });
+
+    it("needs to have a masked word and render same amount of blank spaces", () => {
```

Comment:
```suggestion
    it("has a masked word and render same amount of blank spaces", () => {
```
Be present. Be one with your tests.

<img src="https://media0.giphy.com/media/65ODDoEMHM5wbGDHWT/giphy.gif"/>